src/frontends/pytorch/src/op_table.cpp

```diff
@@ -930,6 +931,9 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.reflection_pad1d.default", op::translate_reflection_pad_nd_fx},
         {"aten.reflection_pad2d.default", op::translate_reflection_pad_nd_fx},
         {"aten.reflection_pad3d.default", op::translate_reflection_pad_nd_fx},
+        {"aten.replicate_pad1d.default", op::translate_replicate_pad_nd_fx},
+        {"aten.replicate_pad2d.default", op::translate_replicate_pad_nd_fx},
+        {"aten.replicate_pad3d.default", op::translate_replicate_pad_nd_fx},
```

Comment:
I think we should also update `get_supported_ops_ts` table as well.

Reply:
done